### PR TITLE
Fix: fix dex parameter in velaux addon

### PR DESCRIPTION
**Pull request by FogDong**
Signed-off-by: FogDong <dongtianxin.tx@alibaba-inc.com>

### Description of your changes

Fixes kubevela/kubevela#4404

### Checklist

I have:

- [x] Title of the PR starts with type (e.g. `[Addon]` , `[example]` or `[Doc]`).
- [ ] Updated/Added any relevant [documentation](https://kubevela.io/docs/reference/addons/overview) and [examples](../examples).
- [ ] New addon should be put in [experimental](../experimental/addons).
- [ ] Update addon should modify the `version` in `metadata.yaml` to generate a new version.
- [ ] Any changes about [verified](../addons) addons should be tested with CI [script](../test/e2e-test/hack/addon-vela-test.sh).

### CODE REVIEW

**Review by wonderflow — COMMENTED** [review]
change the version

File: addons/velaux/resources/velaux.cue [hunk 1]
```diff
@@ -23,24 +23,26 @@ output: {
...
-					name:  "KUBEVELA_API_URL"
-					value: "apiserver.vela-system:8000"
-				},
-				{
-					name:  "DEX_URL"
-					value: "dex.vela-system:5556"
-				},
-			]
+		if parameter["dex"] != _|_ {
+			if parameter["dex"] == true {
```
Comment (charlie0129) [inline addons/velaux/resources/velaux.cue:27]: What if `dex=false`? In this case, `KUBEVELA_API_URL` won't be set.

File: addons/velaux/resources/parameter.cue [hunk 2]
```diff
@@ -20,5 +20,5 @@ parameter: {
 	// +usage=Specify the names of imagePullSecret for private image registry, eg. "{a,b,c}"
 	imagePullSecrets?: [...string]
 	// +usage=Specify whether to enable the dex
-	dex: *"false" | "true"
+	dex?: bool
```
Comment (charlie0129) [inline addons/velaux/resources/parameter.cue:23]:
```suggestion
	dex: *false | bool
```

File: addons/velaux/resources/velaux.cue (hunk 1 above)
Comment (FogDong) [reply addons/velaux/resources/velaux.cue]: Yes that makes more sense, thanks for the review.

**Review by wonderflow — APPROVED** [review]

**Review by wonderflow — APPROVED** [review]

**Review by barnettZQG — APPROVED** [review]

**Pull request merged**